chromium/pages/cancel/style.css

```diff
@@ -47,8 +47,12 @@ form button{
   text-overflow: ellipsis;
 }
 
-#open-url-button{
-  display: inline-block;
-  float: left;
+#open-url-button, #http-once-button {
```

Comment:
Is there any follow up on the UX redesign project mentioned in https://github.com/EFForg/https-everywhere/issues/17867#issuecomment-489494058 ? We have introduced a lots of new UI elements since the discussion in #16669, it would be great to have further updates on it. 

Sorry for derailing from the PR here.

Reply:
@cschanaj I want to try improving the design but not sure if I will have enough time for that.

Reply:
@pipboy96 I have some opinions on the current design too but I would like to improve it without affecting EFF's plan. Some issues like #18482 could be easily resolve with a grid system as mentioned in #17867 . That's why I am asking for a update on the issue. :smiley:

Reply:
@pipboy96 @cschanaj As far as design, I am leaving that to our internal design team when we wrap up things with Ura Design Labs, who is currently doing great work on UX research and helping to incorporate a design system that makes the most sense.

You can track design updates here for now: https://github.com/uracreative/works/issues/29

Reply:
@zoracon Is there some chat work on this design happens on (besides this issue)?

Reply:
@pipboy96 we discuss with Ura labs through their chat but mainly they are taking formal feedback on the issue. Since they wanted a central place for updates and changes.

Reply:
@zoracon Is there any decision on how we are going to implement the design technically?

Reply:
Still awaiting a response, while I wait I am going to just push up the code amendments I have.